tensorflow/cc/gradients/nn_grad_test.cc

```diff
@@ -47,6 +47,15 @@ class NNGradTest : public ::testing::Test {
     EXPECT_LT(max_error, 1e-4);
   }
 
+  void RunTest(const OutputList& xs, const std::vector<TensorShape>& x_shapes,
+               const OutputList& ys, const std::vector<TensorShape>& y_shapes) {
+    TF_ASSERT_OK(scope_.status());
+    float max_error;
+    TF_ASSERT_OK(
+        ComputeGradientError(scope_, xs, x_shapes, ys, y_shapes, &max_error));
+    EXPECT_LT(max_error, 1e-3);
```

Comment:
Super minor: Lets make this 1e-4 to match the rest of the file. The array grads were 1e-3 because of some issue IIRC.